theme/_variables.scss

```diff
@@ -12,4 +12,58 @@ $main-bg: $breadcrumb-bg; // Main element's background colour (WET override).
 // Sign in / off
 $wb-so-background-color: #fff;
 
+/*Banner*/
+$wb-bnr-background-color: #114f7a; /*Dark Cerulean*/
+$wb-bnr-background-linear-gradient: linear-gradient(#176ca7, #114f7a); /*Lochmara, Dark Cerulean*/
+$wb-bar-background: #474747; /*Charcoal*/
+$wb-bnr-border-bottom: #fff; /*white*/
+$wb-bnr-color: #fff; /*white*/
+
+$wb-lng-glb-mn--pnl-btn-border-left: #777; /*gray*/
+$wb-lng-glb-mn--pnl-btn-a-color: #fff; /*white*/
```

Comment:
These 2 variables contain two consecutive dashes between ``mn`` and ``pnl``. I'd recommend revising those variables' names to only use a single dash at that spot.

Specifically:

* ``$wb-lng-glb-mn--pnl-btn-border-left`` would become ``$wb-lng-glb-mn-pnl-btn-border-left``
* ``$wb-lng-glb-mn--pnl-btn-a-color`` would become ``$wb-lng-glb-mn-pnl-btn-a-color``

Reply:
@EricDunsworth Thank you for your advice. I will correct it.